tests-semanticdb/src/test/scala/scala/meta/tests/semanticdb/MetacScalaLibrary.scala

```diff
@@ -5,33 +5,38 @@ import scala.meta.internal.io.FileIO
 import scala.meta.io.AbsolutePath
 import scala.meta.metac._
 import scala.meta.tests.BuildInfo
-import scala.meta.tests.SkipWindows
-import scala.meta.tests.Slow
 
 import java.io.File
 import java.net.URLClassLoader
 import java.nio.file.Files
 import java.nio.file.Paths
 
 import scala.concurrent.duration
+import scala.util.Properties
```

Comment:
didn't know this was an easy way to check for windows... we keep comparing line separators elsewhere.

Reply:
I always forget about this, but we don't use separators to know whether we are on windows, right? We can still have different line endings irrelevant of the system